### PR TITLE
Prepare next release

### PR DESCRIPTION
**Pull request by sni**
this pull request contains everything required for the next release except the release itself.

### CODE REVIEW

**Comment by jacobbaungard** [comment]
Is it possible to change the commit author on "Reload : Fix defunct" so it is not "root" ?

**Comment by sni** [comment]
sure, i faked the author and rebased the commits.

**Comment by sni** [comment]
The commit also changes the location in the debian and rpm package. I could amend those changes to the initial pkg commit or reword the commit description if you like.

**Comment by jacobbaungard** [comment]
It is not really a major issue, but I feel it would be cleaner to have them in the initial pkg commit instead.

**Comment by sni** [comment]
Ok, i moved the spec and debian/ changes to the first commit and rebased everything.

File: t/900-configparsing.t [hunk 1]
```diff
@@ -15,6 +15,7 @@ my $naemon = "$buildroot/src/naemon/naemon";
 my $etc = "$buildroot/t/etc";
 my $precache = "$buildroot/t/var/objects.precache";
 
+plan skip_all => 'config check won\' run as root user' if $> == 0;
```
Comment (jacobbaungard) [inline t/900-configparsing.t]: Rather than skipping the test we could instead pass the --allow-root argument to the Naemon binary.

**Comment by sni** [comment]
yeah, i thought that as well but i was too lazy :-) its fixed now

**Pull request merged**